globals/strings.gd

```diff
@@ -422,6 +424,10 @@ func get_condition_text(conditions):
 				if len(required_bloom_levels) > 0:
 					bloom_str = " " + tr("(Bloom %s)") % "/".join(required_bloom_levels)
 				text += "If attached to %s%s: " % [get_names([condition["required_member_name"]])[0], bloom_str]
+			"attached_to_has_tags":
+				var inverse_str = "without" if condition.get("inverse") else "with"
+				var tags = condition["required_tags"]
+				text += "If attached to Holomem %s tags %s: " % [inverse_str, "/".join(get_tags_strings(tags))]
```

Comment:
Can you please set this up so it can be tr'd better.
I'm guessing we'll want 2 separate strings for with/without.

Probably "If attached to Holomem with %s" or "If attached to Holomem without %s".